WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsFragment.java

```diff
@@ -704,4 +704,17 @@ private void reattachLocalePickerCallback() {
     public void onLocaleSelected(@NotNull String languageCode) {
         onPreferenceChange(mLanguagePreference, languageCode);
     }
+
+    private void handleEnableDisableOpenWebLinksWithJetpackComponents(Boolean newValue) {
+        try {
+            mOpenWebLinksWithJetpackHelper.enableDisableOpenWithJetpackComponents(newValue);
+        } catch (Exception e) {
+            ToastUtils.showToast(
+                    getActivity(),
+                    (newValue ? R.string.preference_open_web_links_with_jetpack_setting_change_enable_error
+                            : R.string.preference_open_web_links_with_jetpack_setting_change_disable_error),
+                    ToastUtils.Duration.LONG);
+            AppLog.e(AppLog.T.UTILS, "Unable to enable or disable open with Jetpack components ", e);
+        }
+    }
```

Comment:
Since this can fail, wonder if we should `AppPrefs.setIsOpenWebLinksWithJetpack` here based on the success/fail result. We can address this in one of the upcoming PRs in case you think what I said is worth also 🙇 .

Reply:
Absolutely great idea @develric . Changed and pushed in commit [f39db35](https://github.com/wordpress-mobile/WordPress-Android/pull/17514/commits/f39db357a3f4a64438f4adde555699f60ed4ecf8)